PWGJE/Core/JetTaggingUtilities.h

```diff
@@ -275,6 +278,22 @@ int jetOrigin(T const& jet, U const& particles, float dRMax = 0.25)
   return 0;
 }
 
+/**
+ * return geometric sign which is calculated scalar product between jet axis with DCA (track propagated to PV )
+ *
+ * @param collision which is needed external table of collision due to postion X and Y
+ * @param jet
+ * @param track which is needed each DCA_X and Y which is measured in jettaggerhfExtension.cxx
+ */
+template <typename T, typename U, typename V>
+int getGeoSign(T const& collision, U const& jet, V const& track)
+{
+  auto sign = TMath::Sign(1, (track.dcaX() - collision.posX()) * jet.px() + (track.dcaY() - collision.posY()) * jet.py() + (track.dcaZ() - collision.posZ()) * jet.pz());
+  if (sign < -1 || sign > 1)
```

Comment:
what is this if for?

Reply:
/**
 * return geometric sign which is calculated scalar product between jet axis with DCA (track propagated to PV )
    * positive and negative value are expected from primary vertex
    * positive value is expected from secondary vertex
    *
    * @param collision which is needed external table of collision due to postion X and Y
    * @param jet
    * @param track which is needed each DCA_X and Y which is measured in jettaggerhfExtension.cxx
    */